charts/argo/templates/server-deployment.yaml

```diff
@@ -52,7 +52,7 @@ spec:
             httpGet:
               path: /
               port: 2746
-              scheme: HTTP
+              scheme: {{ .Values.server.readinessProbeScheme }}
```

Comment:
Could we change this to `readinessProbe.scheme`.

Reply:
Sure. Should I submit another PR, to make above change?